templates_eventbridge/OrgEventBridge.yaml

```diff
@@ -6,13 +6,16 @@ Metadata:
       - Label:
           default: "Sysdig Settings (Do not change)"
         Parameters:
+          - RoleName        
           - EventBridgeRoleName
           - ExternalID
           - TrustedIdentity
           - EventBusARN
           - Regions
           - OrganizationUnitIDs
     ParameterLabels:
+      RoleName:
+        default: "Role Name (Sysdig use only)"    
```

Comment:
@tmcintire what param name is the FE sending the CSPM role name under? We need to make sure it matches

Reply:
This needs to be CSPMRoleName

Reply:
done. changed the name.